### PR TITLE
 Issue 5956 - After an upgrade the server won't start - nsslapd-connta…  …blesize

### PR DESCRIPTION
**Pull request by jchapma**
Description: Add a test case for issue 5956

Related: https://github.com/389ds/389-ds-base/issues/5956

Reviewed by:

### CODE REVIEW

File: dirsrvtests/tests/suites/basic/basic_test.py [hunk 1]
```diff
@@ -1713,6 +1713,38 @@ def test_dscreate_with_different_rdn(dscreate_test_rdn_value):
...
+    assert inst.status() == True
+
+    inst.stop()
+    assert inst.status() == False
+    assert not dse_ldif.get(DN_CONFIG, attr)
+    dse_ldif.add(DN_CONFIG, attr, attr_value)
+    assert dse_ldif.get(DN_CONFIG, attr)
+
+    inst.start()
+    assert inst.status() == True
```
Comment (tbordaz) [inline dirsrvtests/tests/suites/basic/basic_test.py]: You may check that the message "User setting of nsslapd-conntablesize is disabled,..." is logged in the error and after startup the server can answer simple request (like search base "")

**Review by tbordaz — APPROVED** [review]

File: dirsrvtests/tests/suites/basic/basic_test.py [hunk 2]
```diff
@@ -1713,6 +1713,38 @@ def test_dscreate_with_different_rdn(dscreate_test_rdn_value):
         else:
             assert True
 
+def test_conntablesize_attr_dse(topo):
```
Comment (tbordaz) [inline dirsrvtests/tests/suites/basic/basic_test.py]: This test may be skipped on main and after 2.3

**Comment by droideck** [comment]
I think we usually create the PR for `main` first and then cherry-pick it to older branches. Or do you want this PR in 1.4.3 only? If so, then nevermind.

File: dirsrvtests/tests/suites/basic/basic_test.py (hunk 1 above)
Comment (jchapma) [reply dirsrvtests/tests/suites/basic/basic_test.py]: Done

File: dirsrvtests/tests/suites/basic/basic_test.py (hunk 2 above)
Comment (jchapma) [reply dirsrvtests/tests/suites/basic/basic_test.py]: The test runs on branches 1.4.3 -> 2.2

**Comment by jchapma** [comment]
> I think we usually create the PR for `main` first and then cherry-pick it to older branches. Or do you want this PR in 1.4.3 only? If so, then nevermind.

In this case the issue is just on 1.4.3, we dont have this issue on main branch.

**Pull request merged**